### DISCUSSION TITLE
Something changed in the libraries used for the Custom Datasets lesson (results changed even with Daniel's notebook)

### DISCUSSION THREAD

**Discussion by DimOnGit28** [mrdbourke/pytorch-deep-learning#776] [Q&A]
I tried to reproduce the results (not the exact ones but the loss and acc trends of the models) but it was impossible. Of course, I thought I did something wrong so I tried replicating the exact lines of code on my Jupyter notebook (locally). 
But again my results were VERY low compared to Daniel's Solution video (epoch 50: my train_acc = 0.33, train_acc in his Custom Dataset Solution video on youtube = 1.0000) 

So I tried running his own "04. PyTorch Custom Datasets Exercise Solutions" notebook on Google Collab and the results matched mine and not the ones in the video.  (See screenshots in the comment). 

The only explanation would be that something changed in the PyTorch libraries. Maybe an update of the loss, optim, torch.nn, datasets.ImageFolder, DataLoaders, or others.

@mrdbourke

**Comment by DimOnGit28** [comment]
The "04_pytorch_custom_datasets_exercise_solutions" Notebook today : 
![Screen shoot](https://github.com/mrdbourke/pytorch-deep-learning/assets/154806357/92974061-04a1-48f6-b764-c816c03b9d60)


Daniel's video on the same notebook 1 year ago : 
![Video](https://github.com/mrdbourke/pytorch-deep-learning/assets/154806357/c2a18b75-95fb-494b-9ff1-00c5e092ec25)
